serpentTools/seed.py

```diff
@@ -0,0 +1,105 @@
+"""
+Function to copy an input file N times with N randomly generated seeds
+"""
+
+import os
+from os import path
+from shutil import copy
+import random
+
+from six.moves import range
+
+from serpentTools.messages import error, debug
+
+__all__ = ['seedFiles']
+
+
+def _writeSeed(stream):
+    stream.write('\nset seed {}'.format(random.getrandbits(32)))
```

Comment:
I would let the user choose the number of bits used for the random seed.

Reply:
Done. User can specify the length of the digit and the number of bits required is determined